src/apps/material/helper.ts

```diff
@@ -107,6 +107,23 @@ export const getManifestationLanguages = (manifestation: Manifestation) => {
   );
 };
 
+export const getManifestationLanguageIsoCode = (
+  manifestation: Manifestation[]
+) => {
+  const uniqueLanguages = uniqBy(
+    manifestation.map((m) => m.languages),
+    "main[0].isoCode"
+  );
```

Comment:
Please reconsider this approach to determining unique languages.

As I see it is will not return unique `Language` object, it will return unique `Languages` objects based on the iso code of the first main language.

I think a more correct approach would be to return unique `Language` objects that are defined as main languages for the manifestations.